### PR TITLE
List Doichain (DOI)

### PR DESCRIPTION
**Pull request by NiKrause**
- Official project URL: https://www.doichain.org/
- Official block explorer URL: https://explorer.doichain.org/

### CODE REVIEW

File: assets/src/main/java/bisq/asset/coins/Doichain.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+package bisq.asset.coins;
+
+import bisq.asset.Coin;
+import bisq.asset.DefaultAddressValidator;
+
+public class Doichain extends Coin {
+
+    public Doichain() {
+        super("Doichain", "DOI", new DefaultAddressValidator());
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/coins/Doichain.java]: Do not use deprecated validator.

**Review by blabno — CHANGES_REQUESTED** [review]
NACK percomments

File: assets/src/main/java/bisq/asset/coins/Doichain.java (hunk 1 above)
Comment (NiKrause) [reply assets/src/main/java/bisq/asset/coins/Doichain.java]: we are strongly based on Namecoin, so I used Namecoin as an example. This probably means this PR gets rejected. So I'll look again to find an up-to-date validator.

**Comment by blabno** [comment]
It's not rejected. Adjust the code and amend commit.

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by NiKrause** [comment]
@blabno I ammended my commit locally, put couldn't push it without a new pull and naturally got a merge conflict. I resolved the merge conflict (of my old commit and the new commit), committed again and pushed again. I apologize if that's not how you were proposing it, but in case you can still use my new corrected commit, please do so!

**Comment by blabno** [comment]
Code looks ok. Please squash commits into a single one. If you don't know how to do it, add me as collaborator to your fork and I'll do that.

**Pull request closed without merging**